plugins/BEdita/API/tests/TestCase/Controller/Model/SchemaControllerTest.php

```diff
@@ -103,4 +104,26 @@ public function testJsonSchemaAbstractType()
         $this->assertContentType('application/schema+json');
         static::assertFalse($result);
     }
+
+    /**
+     * Test ETag response header and Not Modified response.
+     *
+     * @return void
+     *
+     * @covers ::jsonSchema()
+     */
```

Comment:
`@covers ::generate()` too?

Reply:
Since it's a controller test case we better test only `SchemaController` methods.
`JsonSchema::generate()` is tested in its own `JsonSchemaTest` class.